recipes-extended/onie-tools/onie-tools/onie-bisdn-rescue

```diff
@@ -27,6 +27,11 @@ while getopts "hy" o; do
   esac
 done
 
+if [[ $EUID -ne 0 ]]; then
```

Comment:
Usually I run this command as the basebox user with sudo, but here we require the user to switch user to root.
Should we also allow this command to be run with sudo?

The error message is perfectly clear though, in case we want to keep it like this. (i.e. please run as root)

Reply:
it is checking the EUID, the "Effective User ID". So running with sudo works, since you are effectively root, and this check will succeed.

Reply:
Misunderstanding from my side, looks good